cs/src/io/unsafe/MemoryHelper.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.ConstrainedExecution;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bond.IO.Unsafe
+{
+    internal unsafe class MemoryHelper
+    {
+        [DllImport("msvcrt.dll", EntryPoint = "memcpy", CallingConvention = CallingConvention.Cdecl, CharSet = CharSet.Unicode, SetLastError = false)]
```

Comment:
Another option that was suggested is to use `Marshal.Copy` since what we really need here is copy from unmanaged memory into `byte[]`. 

Also for completeness: I have learned that Mono does have a DllMap feature allowing to remap `DllImport` via configuration. This would actually make it possible to import `memcpy` from different libraries on different platforms. But that is a rather unwieldy thing to maintain so I don't think we should do it here.